.github/workflows/run_admission_test.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           set -x
           cd ./noobaa-operator
           kubectl get events --sort-by='.metadata.creationTimestamp' -A > logs_kubectl_events.txt
-          ./build/_output/bin/noobaa-operator diagnose --db-dump --dir=admission-tests-logs -n test
+          ./build/_output/bin/noobaa-operator diagnostics collect --db-dump --dir=admission-tests-logs -n test
```

Comment:
old `diagnose` command should still work, right? 
shouldn't it be changed to `diagnostics db-dump`?

Reply:
Yes, `diagnose` and also` db-dump` commands should still work with a deprecated printing:
![Screenshot 2023-04-18 at 13 51 38](https://user-images.githubusercontent.com/57721533/232755924-4a13c3ae-5e34-4cd7-b687-131ec634f68d.png)

In `nb diagnose` there is a flag of `db-dump` which I didn't touch and we can still use it when running `diagnostics collect --db-dump`
![Screenshot 2023-04-18 at 13 56 40](https://user-images.githubusercontent.com/57721533/232756972-6a30cc1a-5a00-4d7e-8e60-4d472cc95b4b.png)